src/type.cpp

```diff
@@ -2625,6 +2625,17 @@ llvm::DIType *ReferenceType::GetDIType(llvm::DIScope *scope) const {
                : m->diBuilder->createReferenceType(llvm::dwarf::DW_TAG_reference_type, diTargetType);
 }
 
+bool lIsInsideStdlib(const SourcePos &pos) {
+    std::string filename = pos.name;
+    std::string files[] = {"core.isph", "builtins.isph", "svml.isph", "stdlib,isph", "stdlib.ispc"};
```

Comment:
Hmm, I'd use more unique names for header files which would definitely not be used by user.